android/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
   dependencies {
     classpath 'com.android.tools.build:gradle:3.2.1'
     //noinspection DifferentKotlinGradleVersion
-    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
If you check LINE 3 of this file, it is
```
def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['ReactNativeWebView_kotlinVersion']
```

Our project did *not* define `kotlinVersion`, so it cannot find it.

### Solution

See this [article](https://stackoverflow.com/questions/66834977/wheres-rootproject-ext-in-react-native-project). I believe you can define `kotlinVersion` in *our* project (*not* `react-native-webview` project) and then make the `rootProject.ext.get('kotlinVersion')` to return `1.7.10`, and then you *don't need* this customized codes on `react-native-webview`.

Again, you can *purge* this `react-native-webview` repository!